<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MP4视频播放器--测试</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/new/prefix1/images/11.jpg</normaloff>:/new/prefix1/images/11.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QMyselfVideoWidget" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>591</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>591</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButton_open">
         <property name="text">
          <string>打开</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/new/prefix1/images/1.GIF</normaloff>:/new/prefix1/images/1.GIF</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_player">
         <property name="text">
          <string>播放</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/new/prefix1/images/4.bmp</normaloff>:/new/prefix1/images/4.bmp</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_pause">
         <property name="text">
          <string>暂停</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/new/prefix1/images/5.bmp</normaloff>:/new/prefix1/images/5.bmp</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_stop">
         <property name="text">
          <string>停止</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/new/prefix1/images/6.bmp</normaloff>:/new/prefix1/images/6.bmp</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_volume">
         <property name="text">
          <string>音量</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/new/prefix1/images/volumn.bmp</normaloff>:/new/prefix1/images/volumn.bmp</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_volumeSpeed">
         <property name="maximum">
          <number>99</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_name">
         <property name="text">
          <string>视频名称：</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_position">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_speed">
         <property name="text">
          <string>视频播放进度</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QMyselfVideoWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">qmyselfvideowidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
